aptos-move/framework/src/release_builder.rs

```diff
@@ -63,9 +59,6 @@ impl ReleaseOptions {
             let relative_path = path_relative_to_crate(package_path.join("sources"));
             source_paths.push(relative_path.display().to_string());
         }
-        let bundle = ReleaseBundle::new(released_packages, source_paths);
-        std::fs::create_dir_all(&output.parent().unwrap())?;
-        std::fs::write(&output, bcs::to_bytes(&bundle)?)?;
```

Comment:
This is used for genesis, it needs to be kept

Reply:
updated